src/pages/TermsOfService.js

```diff
@@ -314,12 +315,15 @@ const TermsOfServicePage = () => {
     }
   }
 
-  return div({ style: { padding: '1rem', minHeight: '100%', display: 'flex', justifyContent: 'center', alignItems: 'center' } }, [
+  return div({ role: 'main', style: { padding: '1rem', minHeight: '100%', display: 'flex', justifyContent: 'center', alignItems: 'center' } }, [
     backgroundLogo,
-    div({ style: { backgroundColor: 'white', borderRadius: 5, width: 800, padding: '2rem', boxShadow: Style.standardShadow } }, [
-      div({ style: { color: colors.dark(), fontSize: 38 } }, ['Terra Terms of Service']),
+    div({ style: { backgroundColor: 'white', borderRadius: 5, width: 800, maxHeight: '100%', padding: '2rem', boxShadow: Style.standardShadow } }, [
+      h1({ style: { color: colors.dark(), fontSize: 38, fontWeight: 400 } }, ['Terra Terms of Service']),
       needToAccept && div({ style: { fontSize: 18, fontWeight: 600 } }, ['Please accept the Terms of Service to continue.']),
-      div({ style: { maxHeight: 400, overflowY: 'auto', lineHeight: 1.5, marginTop: '1rem', paddingRight: '1rem' } }, [
+      div({
+        ref: scrollRef,
+        style: { height: '50vh', overflowY: 'scroll', lineHeight: 1.5, marginTop: '1rem', paddingRight: '1rem' }
```

Comment:
Where does the 50vh come from? And why does 'auto' not work for the overflow here? I would expect/hope to avoid use of 'scroll' because it means when the screen is large enough, you still have an empty scrollbar.

Reply:
Basically, I was just trying to get more text into the panel and make it clear that the user must scroll, while not changing the look and feel too much. Macs don't show the scrollbar until you start scrolling, so the first time I saw this page it wasn't obvious that there was more to read.

I played around with adding a button to automatically scroll down a page, (and to indicate that scrolling is necessary) but this was complicated and didn't really seem to help the situation much so I pulled that back out. Hitting the spacebar key does the same thing without needing to do any scroll state synchronization.

`auto` vs `scroll`: yeah, you're right, auto would be fine. On the other hand, the ToS text is baked into the source code and is WAY longer than one page, so I can't imagine there would ever be a scenario where the text fits on a single page without scrolling, even now that the panel can get bigger than 400px. Mostly I was trying to see if I could force MacOS to render the scrollbars. It didn't make a difference, so I could change it back if you prefer. Technically that `scrollRef` isn't necessary anymore either.

`50vh`: I tested a couple of different values relative to the viewport height, and this one seemed to work pretty well for both small and large browser windows. At moderate screen sizes it is fairly similar to how it looks pinned to `400px`, but it allows the panel to expand with taller screens, and it doesn't collapse too much in really short browser windows. At "50% of the viewport height", the window itself has to be something like 300px or less to get a double scrollbar on the text region. Not perfect, but since the page won't get narrower than 1200px anyway that seemed reasonable. It doesn't appear that any of the pages are intended to be mobile-responsive.